CIP-0072/version_1_onchain.json

```diff
@@ -57,49 +57,12 @@
           "required":[
              "action"
           ]
-       },
```

Comment:
Signature has been removed since every cardano transaction has it own signature. The transaction signature (or called Witness) can be used to prove the owner in case of a dApp update or deregistration.